src/main/java/command/PrintAreaLocationsCommand.java

```diff
@@ -0,0 +1,54 @@
+package command;
+
+import eventlist.EventList;
+import locationlist.BusStopList;
+import locationlist.LocationList;
+import storage.Storage;
+import ui.UI;
+
+/**
+ * Represents the command for printing locations available at some areas in NUS.
+ */
+public class PrintAreaLocationsCommand extends Command {
+    String area;
+
+    public PrintAreaLocationsCommand(String area) {
+        this.area = area;
+    }
+
+    /**
+     * Purely print the available locations at the area selected by the user.
+     *
+     * @param events    the list of Events.
+     * @param locations the list of Locations.
+     * @param busStops  the list of BusStops.
+     * @param ui        do outputs.
+     * @param storage   store the data.
+     */
+    @Override
+    public void execute(EventList events, LocationList locations, BusStopList busStops, UI ui, Storage storage) {
+        switch (area.toUpperCase()) {
+        case "FOS":
+            ui.printFosLocations();
+            break;
+        case "FOE":
+            ui.printFoeLocations();
+            break;
+        case "FASS":
+            ui.printFassLocations();
+            break;
+        case "SDE":
+            ui.printSdeLocations();
+            break;
+        case "SOC":
+        case "BIZ":
+            ui.printSocAndBizLocations();
+            break;
+        case "HOSTEL":
+            ui.printHostelLocations();
+            break;
```

Comment:
You may consider adding zones that enable the users to view what are the buildings inside each zone or check the zone of a certain building/faculty/hostel. For now all data are within the Kent Ridge Campus, you may add in more information about the Botanic Campus (The Law School) and all that. You can also include other buildings/facilities like uhc/YIH.